### ISSUE TITLE
JSNM - Collection Portal Request

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/new-collections#234]
Institution contact requests collection portal creation. Please use the form at [Reports/CollectionCreationRequest](http://arctos.database.museum/new_collection.cfm)

**Comment by Jegelewicz** [comment]
> Karly,
> Great! I have sent the MOU to Chris for his signature and will send you a final copy once that is done.
> Now we need to gather the information to set up your portal and collections. I find that it is easier if we transition our conversation to GitHub so that other Arctos community members cab help out as well. If you haven't already gone through the GitHub and sharing information, please do so now, set yourself up with a GitHub account and let me know your username so I can add you to the Arctos team.
> GitHub for Arctos
> 
> If you want information about how the Arctos Community communicates, read Getting Started with Github for Arctos and The Issue Process: How to Make Arctos Work for You. We use GitHub to ask questions of the Arctos community, resolve database questions, and continue development of Arctos. Participation in the various Arctos GitHub repositories will make your Arctos experience much more fulfilling. Questions specific to your data migration should be posted in the ArctosDB/data-migration repository. Questions about how Arctos works, or requestes for new features in Arctos should be posted to the ArctosDB repository.
> To learn more about how data is shared in Arctos, read Sharing Data and Resources. It is also good to understand how data is stored and connected in the Arctos Ecosystem. Read Understanding Specimen Records.
> Also, here is a repeat of my previous message about setting up your collection in Arctos:
> We should start working on the information needed to set up you collection portals on Arctos. Step one is to agree on your institutional acronym. This will be the first part of your three part catalog numbers. I don't find any occurrences using the institutional acronym JSNM in GBIF so if that works for you, I think it is a good choice.
> Next we need to know what collections you want to set up in your portal. If your entire collection is 4,500 objects, you could consider setting up a single collection for everything, however, most institutions set up separate collections based upon taxonomy or specific object areas. Based upon our previous discussions, it appears that setting up two collections, one for fossils and another for eggs might be what you want. We can chat about the pros and cons of a single collection vs. separate ones if you would like.
> Finally, there will be some information needed for each collection once we are ready to set them up. This includes:
> Link to the collection website - probably https://www.ben.edu/museum/Link to the collection loan policy - perhaps a document on the site above or a new page
> Name, email and Arctos username of the person who will manage the Arctos collection - anyone who will be working with your collection data in Arctos should go ahead and create a user account following the instructions in this document.
> License for the data - you can see the choices here.
> Brief description of the collection
> For an example of how most of this information will be used, see the Denver NHM portal page and click the links related to any one of their collections.
> 
> You can read more about these decisions here. Let me know if you have any questions. Once we get you on GitHub, we can work through the decisions there.
> Adios,

Teresa J. Mayfield-Meyer

**Comment by Jegelewicz** [comment]
> Hello Teresa,
> 
> My user name on GitHub is ktumminello and our collection manager’s is cefilipek.
> 
> I would like us to use the institutional acronym JSNM for our collection. I believe we would like the portal set up for two collections, one for fossils and one for eggs.
> 
> We currently do not have our policies on a webpage, however, I can do that next week so there are links directly to the policies. I will look into the examples you provided and I will get back to you with the remaining information next week.
> 
> Thank you!

Karly

**Comment by Jegelewicz** [comment]
> Hi Teresa,
> 
> Please see below for the additional information needed.
> 
>  
> 
> Link to the collection website - https://www.ben.edu/museum/
> 
> Link to the collection loan policy – I will create a webpage/link to this shortly. For now we do not have this available.
> 
> Name, email and Arctos username of the person who will manage the Arctos collection – Colleen Filipek, cfilipek@ben.edu , cfilipek
> 
> License for the data - http://www.ben.edu/information-technology/upload/Acceptable-Use-of-Technology-Resources.pdf
> 
> Brief description of the collection –
> 
> Jurica-Suchy Nature Museum at Benedictine University egg specimens
> 
> Jurica-Suchy Nature Museum at Benedictine University fossil specimens

Karly

**Comment by Jegelewicz** [comment]
> Karly,
> 
> Great timing! I am working on setting up your portal information today. It may take a week or so before the portals are created which will give us time to work on the invoice and getting you familiar with GitHub.
> 
> I will inform Rauthiflor about the W-9 and get them working on your invoice as well.
> 
> I have invited both you and Colleen to join the Arctos GitHub team so that you will be able to post issues there. We will use a GitHub project to manage your progress getting data into Arctos. You can see the project here.
> 
> The first step is to familiarize you with GitHub! In the GitHub for Arctos issue, you will find some instructions that should help you understand what GitHub is and why we use it. Please follow the instructions there and if you have questions about GitHub, post them right there in the issue. One of the great things about GitHub is that the whole Arctos community participates which means that questions get answered more quickly and by the person who has the most experience.
> 
> Also, if you haven't already, create an account in Arctos, but be sure that your username and password meet the following criteria:
> 
>     Go to http://arctos.database.museum and enter a username and password, then select “Create Account” (do not hit the return key or it will think you are trying to login with an account that doesn’t exist yet).
>     Do not select “Create Account” before entering a username and password
>     Username must inlcude only letters and numbers (ASCII characters A-Z, a-z, and 0-9), and cannot begin with a number. Do not include periods in your username (e.g., carla.cicero is not a valid username)
>     Passwords should be at least eight characters, start with a letter, contain only A-Z, a-z, 0-9, and !$%&()`*+,-/:;?_., not contain the username, and consist of at least one letter, number, and special character
>     Once you have created an account and are logged in, from the Arctos main menu select “My Stuff”, then select “Profile”
>     Enter your First, Middle (if applicable, initial ok), and Last names in the appropriate fields
>     Enter your Affiliation (name of your institution)
>     Enter a valid email address. This is important because you cannot be invited as an Operator without a valid email address
>     Select the “Save Profile” button and you are done. However, there are other options that you can set here, including: preferred file format for downloading data; preferred other identifier (e.g., “collector number”) - if you select something here, it will always show up on your Specimen Search page when you are logged in.
> 
> Please have Colleen set up an account as well. Once your accounts are set up, please email your usernames to me.
> 
> We will trade some emails about the invoice, but from now on it will be most efficient if we correspond using GitHub. I'm excited to see you get started as well!
> 
> Adios,

Teresa

**Comment by Jegelewicz** [comment]
> Karly and Colleen,
> I just wanted to let you know that I will be inviting Colleen to become an Arctos operator to day. Please be on the lookout for an email from Arctos as it may end up in junk mail. Follow the instructions in the email to become an Arctos operator. This will allow me to assign manage collection permission to Colleen for your new portals.
> Thanks!
> 

Teresa

**Comment by Jegelewicz** [comment]
@lkvoong I have set JSNM to approve to create collections. Can you please set up their Egg and Paleo collection portals?

Thanks!

**Comment by lkvoong** [comment]
@Jegelewicz 

Hi Teresa,

I've created two new collections for JSNM:

- http://arctos.database.museum/jsnm_egg
- http://arctos.database.museum/jsnm_paleo

I've granted access for the collections as follows:

- jsnm_egg:  Teresa J. Mayfield-Meyer (jegelewicz), Carla (ccicero), Colleen Filipek (cfilipek)
- jsnm_paleo:  Teresa J. Mayfield-Meyer (jegelewicz), Colleen Filipek (cfilipek)

I've also created an institutional portal for JSNM, which is available via the following URL, but it's not yet listed on the Portals page due to caching.  Again, it may take up to 24 hours.
- http://arctos.database.museum/jsnm_all

Thanks,

Lam
